CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# 2019-02-28 v0.1.4
+
+TIFF export:
+
+- Set `writeSequentially` to improve performance.
```

Comment:
Link to PR has proved to be useful e.g. ``[#231](https://github.com/ome/omero-iviewer/pull/231)``